### DISCUSSION TITLE
Database readonly View fields

### DISCUSSION THREAD

**Discussion by gorsar** [serenity-is/Serenity#7278] [Help]
I am getting data from read only fields in the database with view and I want to write data to the detail with master detail relationship. However, although I made the view fields Readonly(true) and the changes are only in the details fields, I am getting an error during save. How can I solve this problem?

**Comment by ademc** [comment]
Yo can try
`[Updateable(false)]`

**Reply by gorsar** [reply]
Thank you this worked..
